lib/mongoid_session_store/session.rb

```diff
@@ -0,0 +1,18 @@
+require 'mongoid'
+
+module MongoidSessionStore
+  class Session
+    include Mongoid::Document
+    include Mongoid::Timestamps
+
+    field :data, type: BSON::Binary, default: -> { Session.binary_data({}) }
+
+    def self.binary_data(data)
+      BSON::Binary.new(Marshal.dump(data), :generic)
+    end
+
+    def self.stats
+      collection.client.command(collstats: storage_options[:collection]).first
+    end
+  end
+end
```

Comment:
The new session object is a very simple `Mongoid::Document` that uses the native `BSON::ObjectId`.